HunterPie/App.xaml.cs

```diff
@@ -129,6 +129,12 @@ private void OnProcessClosed(object sender, ProcessManagerEventArgs e)
             Dispatcher.InvokeAsync(WidgetInitializers.Unload);
             WidgetManager.Dispose();
             Log.Info("{0} has been closed", e.ProcessName);
+            if (e.Process.IsExitedNormally == false 
+                && e.Process.Game == GameProcess.MonsterHunterWorld
+                && ClientConfig.Config.Client.EnableNativeModule)
+            {
+                Log.Info("Game has exited abnormally. If you have not installed Stracker's Loader and CRC bypass mod, turning off \"Enable native module\" in Client Settings may help.", e.ProcessName);
```

Comment:
This will not concatenate `e.ProcessName` to the previous string. Passing more than one argument will call `string.Format(message, ...args)` internally.

Reply:
I've forgotten to add that `{0}` to the template. Added now.